src/lists.py

```diff
@@ -42,17 +45,29 @@ def print_list(x):
 match sys.argv[1]:
     case "mean":
         # put your solution to the first exercise here
-        mean = "mean of x"
+
+        #calculate the sum 
+        sum = 0
+        for i in x:
+            sum += i
+        #calculate the mean
+        
+        mean = (sum/len(x))
         print(mean)
 
     case "times":
         # Put your solution to the second exercise here
         times_three = []
+        for i in x:
+            times_three.append(i*3)
```

Comment:
This is another place where a list comprehension is nifty.

```python
times_three = [3 * i for i in x]
```

would get the job done. But it works exactly the same way as the `for`-loop, and your solution is just as good (and the one I had in mind when writing the exercise).